news/2020-03-13-featured-artist-beatmap-updates-from-the-mappers-guild.md

```diff
@@ -0,0 +1,439 @@
+
+The **osu!** version of this quest was completed by **[Dendy](https://osu.ppy.sh/users/4589786)**, **[Daycore](https://osu.ppy.sh/users/5596337)** and **[piroshki](https://osu.ppy.sh/users/7645522)**
+
+- [Native Construct - Come Hell or High Water](https://osu.ppy.sh/beatmapsets/1051167#osu/2196830) by [Daycore](https://osu.ppy.sh/users/5596337)
+- [Native Construct - Chromatic Abberation](https://osu.ppy.sh/beatmapsets/1062098#osu/2223964) by [piroshki](https://osu.ppy.sh/users/7645522)
+- [Native Construct - Chromatic Lights](https://osu.ppy.sh/beatmapsets/1077275#osu/2254289) by [Daycore](https://osu.ppy.sh/users/5596337)
+
+The **osu!taiko and osu!mania** version of this quest was completed by **[Faputa](https://osu.ppy.sh/users/845733)**, **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)** and **[FAMoss](https://osu.ppy.sh/users/7707789)**
+
+- [Native Construct - Mute](https://osu.ppy.sh/beatmapsets/1053262#taiko/2201028) by [Faputa](https://osu.ppy.sh/users/845733) (osu!taiko)
+- [Native Construct - Come Hell or High Water](https://osu.ppy.sh/beatmapsets/1055375) by [FAMoss](https://osu.ppy.sh/users/7707789) (osu!mania)
+- [Native Construct - Chromatic Abberation](https://osu.ppy.sh/beatmapsets/1080164) hosted by [Cynplytholowazy](https://osu.ppy.sh/users/3901754) (osu!taiko)
+
+![Nekrogoblikon header](https://assets.ppy.sh/artists/53/header.jpg)
+
+For the **Nekrogoblikon mini-pack (osu!)** quest, the mappers had to create and rank at least 3 mapsets of songs by Nekrogoblikon
+
+This quest was completed by **[LMT](https://osu.ppy.sh/users/7262798)**, **[Icekalt](https://osu.ppy.sh/users/5410645)**, **[Mun](https://osu.ppy.sh/users/6699165)**, **[Hectic](https://osu.ppy.sh/users/6885942)**, **[Nao Tomori](https://osu.ppy.sh/users/5364763)** and **[SILENCE PLAYER](https://osu.ppy.sh/users/6381153)**
+
+- [Nekrogoblikon - Mold](https://osu.ppy.sh/beatmapsets/1061153#osu/2221627) hosted by [Icekalt](https://osu.ppy.sh/users/5410645)
+- [Nekrogoblikon - The Skin Thief](https://osu.ppy.sh/beatmapsets/1061535) hosted by [LMT](https://osu.ppy.sh/users/7262798)
+- [Nekrogoblikon - Dragons](https://osu.ppy.sh/beatmapsets/1083931#osu/2266883) by [Nao Tomori](https://osu.ppy.sh/users/5364763)
+
+![Panda Eyes header](https://assets.ppy.sh/artists/55/header.jpg)
+
+For the **On My Own (osu!taiko)** quest, the mappers had to create and rank beatmap for "On My Own" by Panda Eyes
+
+This quest was completed by **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)** and **[Faputa](https://osu.ppy.sh/users/845733)**
+
+- [Panda Eyes - On My Own](https://osu.ppy.sh/beatmapsets/1065837) hosted by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)
+
+For the **Panda Eyes solo** quests, each mapper had to create and rank 1 mapset in their mode of any song by Panda Eyes
+
+The **osu!catch** version of this quest was completed by **[Nelly](https://osu.ppy.sh/users/4741164)**
```

Comment:
when multiple modes have completed the same quest, it'd make more sense to order them in the way people expect (osu, taiko, catch, mania)